src/core/components/footer/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import { Footer } from '../../../common/ui-kit'
```

Comment:
I'd recommend to setup absolute path mapping (new branch, standalone PR) otherwise this going to be horrible.
https://medium.com/hackernoon/absolute-imports-with-create-react-app-4c6cfb66c35d